plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadata.java

```diff
@@ -243,10 +242,6 @@ private JdbcTableHandle flushAttributesAsQuery(ConnectorSession session, JdbcTab
                 return Optional.empty();
             }
 
-            while (columnByName.containsKey(SYNTHETIC_COLUMN_NAME_PREFIX + syntheticNextIdentifier)) {
-                syntheticNextIdentifier++;
-            }
-
             String columnName = SYNTHETIC_COLUMN_NAME_PREFIX + syntheticNextIdentifier;
```

Comment:
`(syntheticNextIdentifier++)`?

Reply:
i tend to avoid pre-/post-increments within expressions.